jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarProcessor.java

```diff
@@ -145,6 +149,42 @@ public static JarType determineJarType(Path jarPath) throws IOException {
     return layers;
   }
 
+  /**
+   * Compute the entrypoint for a standard jar in exploded mode.
```

Comment:
I remember this used to be caught by the formatter. Maybe not anymore.

```suggestion
   * Computes the entrypoint for a standard jar in exploded mode.
```